src/components/ButtonBar.js

```diff
@@ -3,28 +3,21 @@ import { div } from 'react-hyperscript-helpers'
 import { buttonPrimary, buttonSecondary } from 'src/components/common'
 
 
-const ButtonBar = props => {
-  const { onCancel, cancelText, onOk, okText, style } = props
-
-  return div({ style }, [
-    onCancel ? buttonSecondary({
+const ButtonBar = ({ onCancel, cancelText = 'Cancel', onOk, okText = 'Ok', ...props }) => {
+  return div(props, [
+    !!onCancel && buttonSecondary({
       style: { marginRight: '1rem' },
       onClick: onCancel
-    }, [cancelText]) : null,
-    buttonPrimary({ onClick: onOk }, okText)
+    }, [cancelText]),
+    buttonPrimary({ onClick: onOk }, [okText])
   ])
 }
 
-ButtonBar.defaultProps = {
```

Comment:
I am ok with inlining the defaults, but in floatingActionButton the defaults are being set by using "defaultProps"
Are we looking to standardize on one style? If not, when should we use one over the other?

Reply:
For class components, the only option is to use `defaultProps`. For function components, either will work, but inlining the defaults in the arguments is preferred.